spec/rails_helper.rb

```diff
@@ -1,6 +1,7 @@
 ENV['RAILS_ENV'] ||= 'test'
 
 require 'simplecov'
+require 'rspec/rebound'
```

Comment:
What's this doing?

Reply:
So I've not invested the time to investigate these but it'll retry specific failures a few times in severely flaky codebases.

At some point I'd like to remove it and actually understand the flaky tests